Update main readme and examples readme (#192)
grammarly/grammarly-for-developers: examples/readme.md

# Grammarly for Developers Code Examples

> This directory contains numerous code examples you can use as you learn about the Grammarly Text Editor SDK. Most of the examples contain links to CodeSandboxes, so you can tinker with the code in your browser and see what happens.

Reviewer: ```suggestion
This directory contains numerous code examples you can use as you learn about the Grammarly Text Editor SDK. Most of the examples contain links to CodeSandbox—an open-source online code editor—so you can tinker with the code in your browser and better understand our features and functionality.
```

We've organized the examples in two ways: [by Text Editor SDK feature](#code-examples-by-sdk-feature) and [by the technology used](#code-examples-by-the-technology-used).

> > Note: The examples in this repository are created and maintained alongside [the latest major version](https://developer.grammarly.com/docs/changelog/) of the Text Editor SDK. For previous major versions, please reference the following release branches, which are maintained for historical purposes.

Reviewer: ```suggestion
> **Note:** The examples in this repository are created and maintained alongside [the latest major version](https://developer.grammarly.com/docs/changelog/) of the Text Editor SDK. For previous major versions, please reference the following release branches, which are maintained for historical purposes.
```

   - [Activation Example App](./editor-sdk-activation/)

### Autocomplete
This app shows how you can turn on [autocomplete](https://developer.grammarly.com/docs/autocomplete) for your app.

   - [Autocomplete Example App](./editor-sdk-autocomplete/)

### Document dialect
This app shows how you can configure the [document dialect](https://developer.grammarly.com/docs/customization#english-dialect) for your app.

   - [Document Dialect Example App](./editor-sdk-document-dialect/)

### Document domain
This app shows how you can configure the [document domain](https://developer.grammarly.com/docs/v1.x/api/editor-sdk/domain) for your app. 

   - [Document Domain Example App](./editor-sdk-document-domain/)

### Electron
These apps show how to [add the Text Editor Plugin to Electron apps](https://developer.grammarly.com/docs/editor-sdk-electron).

   - [Bundled JavaScript (npm) & Electron Example App](./electron-javascript-npm)
   - [JavaScript & Electron Example App](./electron)
   - [React & Electron Example App](./electron-react/)
   - [Vue & Electron Example App](./electron-vue)

> ### Events (e.g. session stats and document stats)

Reviewer: ```suggestion
### Events
```
Moving to paragraph.

> These apps show how your app can respond to events in the Text Editor SDK.

Reviewer: ```suggestion
These apps show how your app can respond to events (e.g., [`SessionStats`](https://developer.grammarly.com/docs/api/editor-sdk/sessionstats#sessionstats), [`DocumentStats`](https://developer.grammarly.com/docs/api/editor-sdk/documentstats#documentstats)) in the Text Editor SDK.
```

   - [Event Listener Example App (JavaScript)](./editor-sdk-events/)
   - [Stats Example App (React)](./editor-sdk-react-stats/)

### Introductory text
This app shows how you can configure the [introductory text](https://developer.grammarly.com/docs/customization#introductory-text) that is displayed for new users in suggestion cards.

   - [Intro Text Example App](./editor-sdk-intro-text/)

> ### Rich text editors

Reviewer: ```suggestion
### Rich-text editors
```

Author: I'm going to leave this one as-is since this is the way we label them in the docs. I'm totally cool if you want to change it everywhere
https://developer.grammarly.com/docs/editor-sdk-intro

> These apps show how you can add the Text Editor Plugin to various rich text editors. 

Reviewer: ```suggestion
These apps show how you can add the Text Editor Plugin to various rich-text editors. 
```

>    - [CKEditor & JavaScript Example App](./editor-sdk-ckeditor/)
>    - [CKEditor & JavaScript Example App (Imperative)](./editor-sdk-ckeditor-imperative/)
>    - [CKEditor & React Example App](./editor-sdk-react-ckeditor/)
>    - [CKEditor & Vue Example App](./editor-sdk-vue-ckeditor/)
>    - [Quill & JavaScript Example App](./editor-sdk-quill/)
>    - [Quill & JavaScript Example App (Imperative)](./editor-sdk-quill-imperative/)
>    - [Slate & React JavaScript Example App](./editor-sdk-react-slate/)
>    - [TinyMCE & JavaScript Example App](./editor-sdk-tinymce/)
>    - [TinyMCE & JavaScript Example App (Imperative)](./editor-sdk-tinymce-imperative/)
>    - [TinyMCE & React Example App](./editor-sdk-react-tinymce/)
>    - [TinyMCE & Vue Example App](./editor-sdk-vue-tinymce/)

Reviewer: ```suggestion
   - **CKEditor:**
     - [JavaScript Example App](./editor-sdk-ckeditor/)
     - [JavaScript Example App (Imperative)](./editor-sdk-ckeditor-imperative/)
     - [React Example App](./editor-sdk-react-ckeditor/)
     - [Vue Example App](./editor-sdk-vue-ckeditor/)
   - **Quill:**
     - [JavaScript Example App](./editor-sdk-quill/)
     - [JavaScript Example App (Imperative)](./editor-sdk-quill-imperative/)
   - **Slate:** 
     - [React JavaScript Example App](./editor-sdk-react-slate/)
   - **TinyMCE:**
     - [JavaScript Example App](./editor-sdk-tinymce/)
     - [JavaScript Example App (Imperative)](./editor-sdk-tinymce-imperative/)
     - [React Example App](./editor-sdk-react-tinymce/)
     - [Vue Example App](./editor-sdk-vue-tinymce/)
```

### Suggestion categories
This app shows how to configure suggestion categories for your app.

   - [Suggestion Categories Example App](./editor-sdk-suggestions-config/)

### Tone detector
This app shows how to turn on the [tone detector](https://developer.grammarly.com/docs/tone) for your app. 

   - [Tone Detector Example App](./editor-sdk-tone)

### Trusted Authentication
These examples show how you can configure [trusted authentication](https://developer.grammarly.com/docs/trusted-authentication). 
   - [Assertion Server (Java)](./assertion-endpoint-java/)
   - [Assertion Server (Node)](./assertion-endpoint-node/)
   - [Trusted Authentication Example App](./trusted-auth/)

### Turn off UI elements
These apps show three ways to turn off UI elements in the Text Editor Plugin.

   - [Turn Off UI Elements Example App (JavaScript)](./editor-sdk-turn-off-ui-elements/)
   - [Turn Off UI Elements Example App (React)](./editor-sdk-react-turn-off-ui-elements/)

## Code examples by the technology used

> The following is a list of code examples organized by the technology the app uses (e.g. programming language or framework).

Reviewer: ```suggestion
The following is a list of code examples organized by the technology the app uses (e.g., programming language or framework).
```
